src/App/ui.ts

```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyles = createGlobalStyle`
```

Comment:
Here we reset some of the default styles applied by browsers. I didn't go for the full CSS reset, but we could consider doing so as well. This would break some existing tags (like `<h1>`), but we might want to style these explicitly anyway, to prevent inconsistencies across browsers. Something to consider for the future.